internal/applicationsnapshot/report.go

```diff
@@ -50,28 +59,27 @@ type componentSummary struct {
 	TotalWarnings   int                 `json:"total_warnings"`
 }
 
-func NewReport(components []Component, reportSummary bool) (string, error, bool) {
-	var j []byte
-	var err error
-
-	report := Report(components)
-	success := report.Success
-
-	if reportSummary {
-		j, err = json.Marshal(report.reportSummary())
-	} else {
-		j, err = json.Marshal(report)
-	}
-
-	if err != nil {
-		return "", err, false
-	}
-
-	return string(j), nil, success
+// hacbsReport represents the standardized HACBS_TEST_OUTPUT format.
+type hacbsReport struct {
```

Comment:
@dirgim fyi

Reply:
Interesting, we have the same struct here (will be updated with the `int` fields soon): https://github.com/redhat-appstudio/integration-service/blob/main/helpers/integration.go#L42
Does it make sense to import it from there?